kornia/augmentation/base.py

```diff
@@ -276,7 +277,7 @@ def transform_tensor(self, input: torch.Tensor) -> torch.Tensor:
 
     def identity_matrix(self, input) -> torch.Tensor:
         """Return 3x3 identity matrix."""
-        return kornia.utils.eye_like(3, input)
+        return kornia.eye_like(3, input)
```

Comment:
we will eventually remove things from top level

Reply:
```eye_like``` is under geometric module than utils.

Reply:
we should move then